test_verbosefilter.py

```diff
@@ -0,0 +1,108 @@
+# -*- python -*-
```

Comment:
If I change the above line to ``#!/usr/bin/env python`` and make the file executable, it runs fine on it's own, but ``unit2`` doesn't seem to find (and run) it.  I'm willing to poke at it some more, or if it's okay with you, I'll just hard-code running this into ``.travis.yml``.

Reply:
Oooohhh, I think I see the problem...rolling up a diff that should get this working with the ``unit2`` test-discovery...

Reply:
I was not aware of the `unit2` command-line invocation. I've just been playing with it and the documentation is inaccurate and unhelpful. I also can't figure out why it's not picking up verbosefilter. I'll keep looking.

Reply:
...hmmm, nope.  How strange.  I'll have to sit and have a think-stink on the throne (or just hard-code it into ``.travis.yml``), since I promised mrunal work on the openshift installer :worried: 

Basically, running ``unit2`` should pick this suite up like it does all the others.  But I'm okay just hard-coding it too.